### DISCUSSION TITLE
[General] <Looking for projects who want to participate in an open-source hackathon!>

### DISCUSSION THREAD

**Discussion by Blumebee** [ethereum-optimism/developers#404] [Show and tell]
### Discussion Topic

GM - at [OnlyDust](https://www.onlydust.com) we organize monthly hacks known as "ODHack" where we select a number of open-source projects looking to onboard new contributors and gain interest in various blockchain ecosystems. Would any projects be interested in participating? The next one will begin May 23rd (to last 10 days), feel free to have a look here regarding our upcoming [ODHack](https://blog.onlydust.com/odhack-4-0-lets-go/). There are no entry fees - let me know if interested and I'll send over details! There is still time to participate.

**Comment by sbvegan** [comment]
This is awesome! Thank you so much for sharing :)
